shfmt.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: shfmt
+  version: 3.7.0
+  epoch: 0
+  description: A shell formatter
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
```

Comment:
go/build adds all the required packages for building go based apps  we can remove them 
```suggestion
```